src/cheri/app-cap-description.adoc

```diff
@@ -1,6 +1,6 @@
 [appendix]
-[#app_cap_description, reftext="Default capability encoding"]
-== Default Capability Encoding in {cheri_base_ext_name}
+[#app_cap_description, reftext="Default capability encoding ({rvy_default_encoding_name})"]
```

Comment:
This probably wants to be `reftext="{rvy_default_encoding_name}"` for similarity with most of our other named extension sections?